### ISSUE TITLE
Issue with AIR SDK 20.0

### ISSUE THREAD

**Issue by j3k0** [nodrock/ANE-Facebook#21]
When trying to update my build-chain to use AIR SDK 20.0, it would compile but I ran into a runtime issue with the Facebook login.

Calling the `Facebook.accessToken` method:

`Type coercion error from com.freshplanet.ane.AirFacebook:FBAccessToken@1234567 to com.freshplanet.ane.AirFacebook.FBAccessToken`

Using the build with the res-facebook folder renaming.

**Comment by nodrock** [comment]
Are you sure you don't have the old ANE in the same folder? Or maybe swc with that class.

**Comment by j3k0** [comment]
Yes sure, this was my issue last time... I won't fall into the same trap twice!

Changing the PATH to mxmlc and adt to AIR SDK 20.0 creates the run-time issue. Changing back to SDK 18.0 fixes it.

I don't have 19.0, if you'd learn something: I can install it and try...?
